### ISSUE TITLE
[bug]: A lot of controls are not labeled for accessibility

### ISSUE THREAD

**Issue by zersiax** [hoppscotch/hoppscotch#2137]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

When I view hoppscotch.io with a screen reader, a lot of buttons and anchor tags on the page have no discernible text, making it impossible to figure out what these controls are for. I expect these to be properly labeled as per WCAG standards so people who don't rely on their eyesight can make use of the tool.  This is easy enough to fix by just setting a textual label on these components, can even be an aria-label which only labels elements for screen readers and won't take up visual space.

### Steps to reproduce

1. Load Narrator/VoiceOver/Orca/NVDA depending on what your OS is.
2. Visit hoppscotch.io.
3. Tab through the controls on the page. This is not how screenreader users generally navigate a page, but it will give a decent idea of just how many elements are unlabeled.

### Environment

Production

### Version

Cloud

**Comment by liyasthomas** [comment]
Hi @zersiax, thank you for opening this issue ticket. I'll be trying our max to improve the screen reader experience and increase the overall site accessibility.

**Comment by sweetkane** [comment]
Hello. I am a senior studying CS at the University of Michigan. I am supposed to make an open source contribution for my Software Engineering class, and I would love to take care of this issue if it hasn't been fixed already. Could you possibly reassign it to me?

**Comment by liyasthomas** [comment]
Hi @sweetkane, you can pick up this issue. Assigning it to you.

**Comment by sweetkane** [comment]
Hello. I have made changes for this issue. Can I have access to create a branch and make a pull request?

**Comment by liyasthomas** [comment]
Hi @sweetkane, if your changes are in your fork of Hoppscotch, you can directly open a pull request to our `main` branch.

**Comment by thgcode** [comment]
Hello!
I thought of creating a new issue, but found this one, I tested hoppscotch with NVDA and it read only "button" on the buttons. If pressing tab it read the button name, but not when pressing the arrow keys. I think it could because the button name is set to "button" and the description set to the button name, that is somehow confusing the screen reader, but some buttons are correct like the body button or the parameters button.
